src/prommis/nanofiltration/costing/diafiltration_cost_model.py

```diff
@@ -440,3 +441,98 @@ def variable_operating_cost_constraint(blk):
                 to_units=blk.costing_package.base_currency
                 / blk.costing_package.base_period,
             )
+
+    def cost_precipitator(
+        blk,
+        precip_volume,
+    ):
+        """
+        Costing method for precipitator unit. Assumes these are horizontal vessels
+        made from 1.25 in thick carbon steel, includes platforms and ladders, and each
+        instance is one unit (default args).
+
+        References:
+            residence time:
+                https://www.sciencedirect.com/science/article/pii/S0304386X19309806
+                https://onlinelibrary.wiley.com/doi/full/10.1002/ceat.201700667
+                https://pubs.acs.org/doi/full/10.1021/acs.iecr.1c04876
+                https://www.sciencedirect.com/science/article/pii/S0304386X01002134
+            vessel dimensions:
+                https://www.accessengineeringlibrary.com/content/book/9781260455410/back-matter/appendix1?implicit-login=true
+
+        Args:
+            precip_volume: volume of the precipitator as calculated by the unit model (m3)
+        """
+
+        # calculate the volume needed
+        blk.volume_capacity = Var(
+            initialize=120,
+            domain=NonNegativeReals,
+            doc="Volume requirement of precipitator vessel",
+            units=units.m**3,
+        )
+
+        # account for a 20% headspace
```

Comment:
Should this value be an argument as well? The default value could be set to 1.2 as done here. I'm just a bit uncertain how conventional this value is. Is it likely that a user would want to change this value. If so, it should be an argument. If not, it still may be worth adding as an argument just in case